phpBB/styles/prosilver/template/ucp_prefs_wysiwyg.html

```diff
@@ -0,0 +1,60 @@
+<!-- INCLUDE ucp_header.html -->
+
+<form id="ucp" method="post" action="{S_UCP_ACTION}"{S_FORM_ENCTYPE}>
+
+<h2>{L_WYSIWYG_TITLE}</h2>
+<div class="panel">
+	<div class="inner">
+
+	<fieldset>
+	<!-- IF ERROR --><p class="error">{ERROR}</p><!-- ENDIF -->
+	<dl>
+		<dt><label for="wysiwyg_default_mode0">{L_WYSIWYG_EDITOR_SELECT}{L_COLON}</label></dt>
+		<dd>
+			<select data-togglable-settings="true" name="wysiwyg_editor">
+				<option data-toggle-setting="#wysiwyg_{{ WYSIWYG_DEFAULT_EDITOR_TITLE }}_settings_container"{% if CURRENT_WYSIWYG is empty %} {% endif %} value="">{L_WYSIWYG_EDITOR_DEFAULT}</option>
+				<!-- BEGIN wysiwyg -->
+				<option data-toggle-setting="#wysiwyg_{wysiwyg.EDITOR_TITLE}_settings_container" value="{wysiwyg.EDITOR_ID}"{% if wysiwyg.EDITOR_ID == WYSIWYG_EDITOR %} selected{% endif %}>{wysiwyg.EDITOR_TITLE}</option>
+				<!-- END wysiwyg -->
+				<!-- EVENT ucp_prefs_wysiwyg_editor_option -->
+			</select>
+		</dd>
+	</dl>
+	<!-- EVENT ucp_prefs_wysiwyg_prepend -->
+	<!-- BEGIN wysiwyg -->
+		<div id="wysiwyg_{wysiwyg.EDITOR_TITLE}_settings_container">
+			<!-- EVENT ucp_prefs_wysiwyg_options_prepend -->
+			<dl>
+				<dt><label>{L_WYSIWYG_DEFAULT_MODE}{L_COLON}</label></dt>
+				<dd>
+					<label><input type="radio" name="wysiwyg_default_mode[{{ wysiwyg.EDITOR_ID }}]" value="{{ constant('\\phpbb\\bbcode\\convert_editor\\base::DEFAULT_WYSIWYG_MODE') }}"{% if wysiwyg.S_WYSIWYG_MODE b-and constant('\\phpbb\\bbcode\\convert_editor\\base::DEFAULT_WYSIWYG_MODE') %} checked{% endif %}{% if not (wysiwyg.WYSIWYG_MODES_AVAILABLE b-and constant('\\phpbb\\bbcode\\convert_editor\\base::DEFAULT_WYSIWYG_MODE')) %} disabled{% endif %}/> {L_WYSIWYG_DEFAULT_MODE_WYSIWYG}</label>
```

Comment:
Does it make sense to have lines this long? Do I need to make them shorter? I may need to add more template logic, depending on the suggestions about the code in the server, though....